amoro-ams/src/main/java/org/apache/amoro/server/scheduler/TimelyTableScheduler.java

```diff
@@ -39,18 +41,32 @@
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
 
-public abstract class BaseTableExecutor extends RuntimeHandlerChain {
+public abstract class TimelyTableScheduler extends RuntimeHandlerChain {
```

Comment:
```suggestion
public abstract class PeriodicTaskScheduler RuntimeHandlerChain {
```
The tasks execute with a fixed period, maybe `PeriodicTaskScheduler`, `FixedTableScheduler` is more fittable?

Reply:
yeah
PeriodicTaskScheduler is better